cypress/e2e/crypto/decryption-failure.spec.ts

```diff
@@ -67,6 +68,7 @@ describe("Decryption Failure Bar", () => {
     let roomId: string;
 
     beforeEach(function () {
+        skipIfRustCrypto();
```

Comment:
is it not possible to have this in the describe block?

Reply:
oh probably

Reply:
might mess up your output possibly, up to you

Reply:
no, because the `cy.state` hack doesn't work outside an actual test.